manifests/v/vim/vim/8.2.3197/vim.vim.installer.yaml

```
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: vim.vim
+PackageVersion: 8.2.3197
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3197/gvim_8.2.3197_x64.exe
+  InstallerSha256: B6930474E6C89D87D6575BB2D5BE9CAE1AA2C7201082A78D10C9D0886F46BBE8
+  InstallerLocale: en-US
+  UpgradeBehavior: install
+- Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3197/gvim_8.2.3197_x86.exe
+  InstallerSha256: 2BB3E1155A92DA1462AF34EE025AC3A059A47C044B0162730A8680E4D13A5EAA
```

Signed version just came out 👍 

```suggestion
  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3197/gvim_8.2.3197_x86_signed.exe
  InstallerSha256: D3579BD7B75A86FE65CD219BE7F711775601364088057270CD64CA3D94FEACE3
```